locales/en/cie.md

```diff
@@ -0,0 +1,9 @@
+**EIC**, the Electronic Identity Card, allows you to access all the services of the Public Administration. You can use it to conveniently access IO on your smartphone. You will also need the pin that is given to you half in the registry and half by mail together with the card.
```

Comment:
```suggestion
**CIE**, the Electronic Identity Card, allows you to access all the services of the Public Administration. You can use it to conveniently access IO on your smartphone. You will also need the pin that is given to you half in the registry and half by mail together with the card.
```
Since it is always referred in italian we should keep the acronym. It could be confusing